### PR TITLE
Oracle Data base documentation updates

### PR DESCRIPTION
**Pull request by v-muuppugund**
Required items, please complete

   
   Change(s):
   - Oracle Data base documentation

   Reason for Change(s):
   - As documentation is not clear and there is a git hub issue (https://github.com/Azure/Azure-Sentinel/issues/8802)

   Version Updated:
   - 3.0.0
   

   Testing Completed:
   - 
image
image

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Solutions/OracleDatabaseAudit/Data Connectors/Connector_OracleDatabaseAudit.json
Comment (v-dvedak) [inline Solutions/OracleDatabaseAudit/Data Connectors/Connector_OracleDatabaseAudit.json:1]: Shouldn't the database be one word?

**Comment by v-sudkharat** [comment]
Hi @v-muuppugund, Please add releasenotes reference in CreateUi file and update same in zip add well. Thanks!

File: Solutions/OracleDatabaseAudit/Data/system_generated_metadata.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  "firstPublishDate": "2021-11-05",
+  "support": {
+    "name": "Microsoft Corporation",
+    "email": "support@microsoft.com",
+    "tier": "Microsoft",
+    "link": "https://support.microsoft.com"
+  },
+  "Data Connectors": "[\n  \"Data Connectors/Connector_OracleDatabaseAudit.json\"\n]",
+  "Parsers": "[\n  \"OracleDatabaseAuditEvent.txt\"\n]",
```
Comment (v-sudkharat) [inline Solutions/OracleDatabaseAudit/Data/system_generated_metadata.json:29]: Please update the parser extension to yaml and update with same in data file.  Once changes added in data file, please repackage the solution.

Comment (v-muuppugund) [reply Solutions/OracleDatabaseAudit/Data/system_generated_metadata.json]: Completed these changes and JFYI, didn't add as this PR mainly  content review once done ,thought of doing these changes,pushed now

**Comment by v-muuppugund** [comment]
> Hi @v-muuppugund, Please add releasenotes reference in CreateUi file and update same in zip add well. Thanks!

As discussed ,JFYI,these changes are checked in,now pushed it.

**Comment by v-sudkharat** [comment]
Hi @v-muuppugund, please add releasenotes reference in CreateUI file - 
image

and update same in zip as well.

**Comment by v-sudkharat** [comment]
The below Highlighted file is inside the package folder, could you please check on that - 
image

**Comment by v-muuppugund** [comment]
> The below Highlighted file is inside the package folder, could you please check on that - image

This came from package its self

**Comment by v-muuppugund** [comment]
> Solution contains the Analytic rule, Hunting query. Please check on createUi file - image
> 
> image
> 
> Same check on MainTemplate as well.

I didn't change any analytic rule and packaged it ,then its sould be earlier issue

**Comment by v-muuppugund** [comment]
> Hi @v-muuppugund, please add releasenotes reference in CreateUI file - image
> 
> and update same in zip as well.

Release notes are missing in the solution its self,I have added,will link it

**Comment by v-muuppugund** [comment]
> Hi @v-muuppugund, Please add releasenotes reference in CreateUi file and update same in zip add well. Thanks!

Updated the Release notes

**Comment by v-muuppugund** [comment]
> > The below Highlighted file is inside the package folder, could you please check on that - image
> 
> This came from package its self image

Removed the test parameters file

**Review by v-sudkharat — DISMISSED** [review]
checking for the testParameters.json file

**Review by v-sudkharat — APPROVED** [review]
Changes looks good.

**Pull request merged**